config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -382,6 +382,27 @@ postsubmits:
         - --project=k8s-testimages
         - --build-dir=.
         - gencred/
+  - name: post-test-infra-gencred-refresh-kubeconfig
+    cluster: test-infra-trusted
+    run_if_changed: '^(config/prow/gencred-config/|config/jobs/kubernetes/test-infra/test-infra-trusted\.yaml)'
+    decorate: true
+    branches:
+    - ^master$
+    spec:
+      serviceAccountName: gencred-refresher
+      containers:
+      - name: gencred
+        image: gcr.io/k8s-prow/gencred:v20220511-a855e15670
```

Comment:
note: I have updated the image tag here with force push, this image reference is a valid ref now